### PR TITLE
Removing Inline styles and created a seperate style sheet

### PR DESCRIPTION
**Pull request by hemant1026**

### CODE REVIEW

File: code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx [hunk 1]
```diff
@@ -8,8 +9,18 @@ import Avatar from '@mui/material/Avatar';
 import Divider from '@mui/material/Divider';
 import Button from '@mui/material/Button';
 import Copyright from 'src/internals/components/Copyright';
+import styles from './Profile.module.css';
```
Comment (faizanurfi) [inline code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx]: Rename the file to Profile.css and import like this. No need to export it as an object `styles`

`import './Profile.css';`

File: code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx [hunk 2]
```diff
@@ -18,61 +29,60 @@ const profileData = {
   phone: '+1 123-456-7890',
 };
 
-
-export const Profile = () => {
+export const Profile: React.FC = () => {
   return (
-    <Box sx={{ width: '100%', maxWidth: { sm: '100%', md: '800px' }, mx: 'auto', p: 3, borderRadius: 2 }}>
-      <CardContent sx={{ textAlign: 'center', mb: 3 }}>
+    <Box className={styles.profileContainer}>
```
Comment (faizanurfi) [inline code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx]: After changing file name and updating import, change this and rest of the classNames to:

`<Box className='profileContainer'>`

File: code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx [hunk 3]
```diff
@@ -18,61 +29,60 @@ const profileData = {
...
+      <CardContent className={styles.cardContentCentered}>
         <Avatar
-          alt={profileData.firstName + ' ' + profileData.lastName}
-          src="/path/to/avatar.jpg"
-          sx={{ width: 120, height: 120, margin: '0 auto', mb: 2 }}
+          src="/code/packages/web/src/pages/dashboard/dashboard_pages/profile/avatar.jpg"
```
Comment (faizanurfi) [inline code/packages/web/src/pages/dashboard/dashboard_pages/profile/Profile.tsx]: Avatar is not being displayed. Plus inline paths can cause errors:

Do this instead:
- Move `avatar.jpg` to `src/assets` folder
- Import img like this `import avatarSrc from 'src/assets/avatar.jpg';`
- Use that here like this `src={avatarSrc}`

File: code/packages/web/src/pages/dashboard/dashboard_pages/profile/avatar.jpg
Comment (faizanurfi) [inline code/packages/web/src/pages/dashboard/dashboard_pages/profile/avatar.jpg:1]: Move this file to `src/assets` folder

**Review by faizanurfi — CHANGES_REQUESTED** [review]

**Review by faizanurfi — APPROVED** [review]

**Review by tushar-kendre — APPROVED** [review]

**Pull request merged**